runtime/Makefile

```diff
@@ -119,7 +119,7 @@ all: runtime
 bin/sledgert: ${CFILES}
 	@echo "Compiling runtime"
 	@mkdir -p bin/
-	${CC} ${INCLUDES} ${CFLAGS} ${LDFLAGS} ${JSMNCFLAGS} -L/usr/lib/ $^ -o bin/sledgert
+	${CC} ${CFLAGS} ${LDFLAGS} ${JSMNCFLAGS} ${INCLUDES}  -L/usr/lib/ $^ -o bin/sledgert
```

Comment:
What difference does this make?

Reply:
I think that I was trying to use one of the debugging flags, and it seemed to not be working as expected. I think I had a hypothesis that the flags we are adding to CFLAGS were not evaluating as expected in our header files. I'm not actually sure if this made an actual difference at this point.

Reply:
Ran a small experiment to validate. The order shouldn't matter. I'll revert the change on this line.